dist/tools/tunslip/tunslip6.c

```diff
@@ -147,18 +158,34 @@ stamptime(void)
 int
 is_sensible_string(const unsigned char *s, int len)
 {
-    int i;
+    if(len > 0) {
+        if (s[0] == 0x60) {
+            /* Possibly IPv6 packet with default traffic class, assume non-printable */
+            return 0;
+        }
+    }
 
-    for (i = 1; i < len; i++) {
+    int ret = 0;
+    for (int i = 1; i < len; i++) {
         if (s[i] == 0 || s[i] == '\r' || s[i] == '\n' || s[i] == '\t') {
             continue;
         }
         else if (s[i] < ' ' || '~' < s[i]) {
             return 0;
         }
+        /* only return 1 if the string contains at least one letter or number */
```

Comment:
The reason that the tunslip application tries to find printable strings is to be able to multiplex debug messages with SLIP traffic. If you want a packet dump you can run tunslip with the switch `-v4` and everything will be dumped to the terminal as well as passed on to the tun interface.